homeassistant/components/lektrico/sensor.py

```diff
@@ -0,0 +1,294 @@
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PARALLEL_UPDATES = 1
+SCAN_INTERVAL = timedelta(seconds=10)
+
+
+@dataclass
+class LektricoSensorEntityDescription(SensorEntityDescription):
+    """A class that describes the Lektrico sensor entities."""
+
+    unit_fn: str | None = None
+    value_fn: Any | None = None
+
+
+SENSORS: tuple[LektricoSensorEntityDescription, ...] = (
+    LektricoSensorEntityDescription(
+        key="charger_state",
+        name="Charger_State",
+        value_fn=lambda x: x.charger_state,
+    ),
+    LektricoSensorEntityDescription(
+        key="charging_time",
+        name="Charging_Time",
+        unit_fn=TIME_SECONDS,
+        value_fn=lambda x: x.charging_time,
+    ),
+    LektricoSensorEntityDescription(
+        key="current",
+        name="Current",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_CURRENT,
+        unit_fn=ELECTRIC_CURRENT_AMPERE,
+        value_fn=lambda x: x.current,
+    ),
+    LektricoSensorEntityDescription(
+        key="instant_power",
+        name="Instant_Power",
+        device_class=DEVICE_CLASS_POWER,
+        unit_fn=POWER_KILO_WATT,
+        value_fn=lambda x: x.instant_power,
+    ),
+    LektricoSensorEntityDescription(
+        key="session_energy",
+        name="Session_Energy",
+        device_class=DEVICE_CLASS_ENERGY,
+        unit_fn=ENERGY_KILO_WATT_HOUR,
+        value_fn=lambda x: x.session_energy,
+    ),
+    LektricoSensorEntityDescription(
+        key="temperature",
+        name="Temperature",
+        state_class=STATE_CLASS_MEASUREMENT,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        unit_fn=TEMP_CELSIUS,
+        value_fn=lambda x: x.temperature,
+    ),
+    LektricoSensorEntityDescription(
+        key="total_charged_energy",
+        name="Total_Charged_Energy",
+        state_class=STATE_CLASS_TOTAL_INCREASING,
+        device_class=DEVICE_CLASS_ENERGY,
+        unit_fn=ENERGY_KILO_WATT_HOUR,
+        value_fn=lambda x: x.total_charged_energy,
+    ),
+    LektricoSensorEntityDescription(
+        key="voltage",
+        name="Voltage",
+        device_class=DEVICE_CLASS_VOLTAGE,
+        unit_fn=ELECTRIC_POTENTIAL_VOLT,
+        value_fn=lambda x: x.voltage,
+    ),
+    LektricoSensorEntityDescription(
+        key="install_current",
+        name="Install_Current",
+        device_class=DEVICE_CLASS_CURRENT,
+        unit_fn=ELECTRIC_CURRENT_AMPERE,
+        value_fn=lambda x: x.install_current,
+    ),
+    LektricoSensorEntityDescription(
+        key="dynamic_current",
+        name="Dynamic_Current",
+        device_class=DEVICE_CLASS_CURRENT,
+        unit_fn=ELECTRIC_CURRENT_AMPERE,
+        value_fn=lambda x: x.dynamic_current,
+    ),
+    LektricoSensorEntityDescription(
+        key="led_max_brightness",
```

Comment:
missing unit of measurement and state class

Reply:
"led_max_brightness" - is an integer from 20 to 100  (it is %) and shows the brightness of the leds of the physical device.
I put unit of measurement to be PERCENTAGE
I put state class to be SensorStateClass.MEASUREMENT